scipy/stats/_distr_params.py

```diff
@@ -105,6 +105,8 @@
     ['truncexpon', (4.6907725456810478,)],
     ['truncnorm', (-1.0978730080013919, 2.7306754109031979)],
     ['truncnorm', (0.1, 2.)],
+    ['truncweibull_min', (2.7866166930421596,
+                          0.243353416557749, 3.791737951698754)],
```

Comment:
This is not the best choice for the right end of the support because the distribution is essentially only left-truncated.

![image](https://user-images.githubusercontent.com/6570539/154893664-c0749935-123f-42ba-b6c3-8634dc5f2d1a.png)

How about:

```suggestion
    ['truncweibull_min', (2.5, 0.25, 1.25)],
```

Reply:
Fixed in next commit.